src/store/bankdata/actions.ts

```diff
@@ -1,50 +1,56 @@
 import { ActionContext } from 'vuex';
-import axios, { AxiosResponse } from 'axios';
-import { BankAccount, BankAccountData, BankAccountRequest, BankAccountResponse } from '@src/view_models/BankAccount';
+import { BankAccount, BankAccountData } from '@src/view_models/BankAccount';
 import { Validity } from '@src/view_models/Validity';
 
 export const actions = {
-	setBankData( context: ActionContext<BankAccount, any>, payload: BankAccountRequest ): Promise<void> {
-		context.commit( 'SET_IS_VALIDATING', true );
-		return axios( payload.validationUrl, {
-			method: 'get',
-			headers: { 'Content-Type': 'multipart/form-data' },
-			params: payload.requestParams,
-		} ).then( ( validationResult: AxiosResponse<BankAccountResponse> ) => {
-			const validity = validationResult.data.status === 'ERR' ? Validity.INVALID : Validity.VALID;
-			context.commit( 'SET_BANK_DATA_VALIDITY', validity );
-			if ( validity === Validity.VALID ) {
-				context.commit( 'SET_BANKNAME', validationResult.data.bankName );
-				context.commit( 'SET_BANKDATA', {
-					accountId: validationResult.data.iban,
-					bankId: validationResult.data.bic,
-				} );
-			} else {
-				context.commit( 'SET_BANKNAME', '' );
-			}
-			context.commit( 'SET_IS_VALIDATING', false );
-		} );
-	},
-	initializeBankData( context: ActionContext<BankAccount, any>, payload: BankAccountData & { bankName: string} ): void {
-		if ( payload.accountId === '' ) {
-			return;
+	initializeBankData( context: ActionContext<BankAccount, any>, payload: BankAccountData ): void {
```

Comment:
Do we really need this? IIR we made the conscious choice to not store the bank data in localstorage (even if we encrypt this) to avoid spooking the donors. This is also the current behavior I observed while user-testing (bank data getting lost on reload). If we keep this behavior, we could drop this code because it will never be executed.

Reply:
After talking to the PM - we do need this, not storing and/or re-displaying the IBAN on reload is an error.